packages/perseus-core/src/widgets/image/index.ts

```diff
@@ -28,6 +31,14 @@ const imageWidgetLogic: WidgetLogic = {
     defaultWidgetOptions,
     supportedAlignments: ["block", "full-width"],
     defaultAlignment: "block",
+    // This widget's accessibility depends on its contents: if the image
+    // has a background but no alt text, it is not accessible
+    accessible: (widgetOptions: PerseusWidgetOptions): boolean => {
+        const imageOptions = widgetOptions as PerseusImageWidgetOptions;
+        const bgImage = imageOptions.backgroundImage;
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+        return !(bgImage && bgImage.url && !imageOptions.alt);
```

Comment:
We can fix the lint warning instead of suppressing it. 

```suggestion
        return !(bgImage?.url != null && !imageOptions.alt);
```

Reply:
Thanks for the suggestion! I agree this is cleaner and avoids disabling lint rules.